pkgs/applications/display-managers/lightdm/default.nix

```diff
@@ -1,33 +1,64 @@
-{ stdenv, fetchurl, pam, pkgconfig, libxcb, glib, libXdmcp, itstool, libxml2
-, intltool, xlibsWrapper, libxklavier, libgcrypt, libaudit, coreutils
+{ stdenv, fetchFromGitHub, pam, pkgconfig, autoconf, automake, libtool, which
+, libxcb, glib, libXdmcp, itstool, libxml2, intltool, xlibsWrapper, libxklavier
+, libgcrypt, libaudit, coreutils, polkit
+, gtk-doc, gnome3
+, gobjectIntrospection
+, vala
 , qt4 ? null
 , withQt5 ? false, qtbase
 }:
 
 with stdenv.lib;
 
-let
-  ver_branch = "1.24";
-  version = "1.24.0";
-in
 stdenv.mkDerivation rec {
-  name = "lightdm-${version}";
+  pname = "lightdm";
+  version = "1.26.0";
 
-  src = fetchurl {
-    url = "${meta.homepage}/${ver_branch}/${version}/+download/${name}.tar.xz";
-    sha256 = "18j33bm54i8k7ncxcs69zqi4105s62n58jrydqn3ikrb71s9nl6d";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "CanonicalLtd";
+    repo = pname;
+    rev = version;
+    sha256 = "1mhj6l025cnf2dzxnbzlk0qa9fm4gj2aw58qh5fl4ky87dp4wdyb";
   };
 
-  nativeBuildInputs = [ pkgconfig intltool ];
+  nativeBuildInputs = [
+    autoconf
+    automake
+    gobjectIntrospection
+    gtk-doc
+    intltool
+    itstool
+    pkgconfig
+    vala
+  ];
+  
   buildInputs = [
-    pam libxcb glib libXdmcp itstool libxml2 libxklavier libgcrypt
-    qt4 libaudit
+    glib
+    gnome3.yelp_tools
+    gnome3.yelp_xsl
+    itstool
```

Comment:
gnome3.yelp_tools gnome3.yelp_xsl itstool libtool which and possibly libxml2 should go to `nativeBuildInputs`

Reply:
i removed libxml2 now. I don't think it needed it.